### PR TITLE
Remove liveness probes for pachd

### PR DESCRIPTION
**Pull request by tybritten**

### CODE REVIEW

File: etc/helm/pachyderm/templates/pachd/deployment.yaml [hunk 1]
```diff
@@ -320,11 +320,6 @@ spec:
             command:
             - /pachd
             - --readiness
```
Comment (jrockway) [inline etc/helm/pachyderm/templates/pachd/deployment.yaml:322]: I think we can keep the pachd liveness probe.

File: etc/helm/pachyderm/values.schema.json [hunk 2]
```diff
@@ -439,6 +439,9 @@
                         "limits": {
                             "type": "object",
                             "properties": {
+                                "cpu": {
+                                    "type": "integer"
```
Comment (jrockway) [inline etc/helm/pachyderm/values.schema.json:443]: This should be a string.  "100m" for example.

**Review by jrockway — CHANGES_REQUESTED** [review]

File: etc/helm/pachyderm/values.schema.json (hunk 2 above)
Comment (tybritten) [reply etc/helm/pachyderm/values.schema.json:443]: yeah weird, that must have been from upstream. fixed

File: etc/helm/pachyderm/templates/pachd/deployment.yaml [hunk 3]
```diff
@@ -325,13 +325,13 @@ spec:
...
+          failureThreshold: 10
+          timeoutSeconds: 30
         startupProbe:
           exec:
             command:
             - /pachd
             - --readiness
-          failureThreshold: 10
```
Comment (jrockway) [inline etc/helm/pachyderm/templates/pachd/deployment.yaml:333]: We still need these.  Basically gives pachd 300 seconds to wait for postgres, apply migrations, and setup the config from the environment.

**Review by jrockway — APPROVED** [review]
1 nit ;)

**Pull request merged**